rtb/verbali/interni/VI_2024-12-13_ver1.0.0.typ

```diff
@@ -0,0 +1,106 @@
+  versioni: (
+    "1.0.0",
+    [15/12/2024],
+    "Davide Martinelli",
+    "Davide Marin",
+    [- Stesura del verbale],
+  ),
+  content: content,
+)
+
+#infoRiunione(
+  data: "13/12/2024",
+  durata: "2 ore",
+  luogo: "Chiamata Discord",
+  ora: "11:00",
+  partecipantiEsterni: (),
+)
+
+= Sintesi
+Dopo il #rifGlossario("SAL") di fine secondo _#rifGlossario("sprint")_ il gruppo si è riunito per fare il punto della situazione e pianificare il lavoro da svolgere nel prossimo _sprint_.
+
+= Ordine del giorno
+Durante la discussione interna sono emerse varie attività da completare nel prossimo _sprint_, principalmente legate alla documentazione.
+
+== Strumenti per il diagramma di Gantt
+Sono state riscontrate delle difficoltà nel trovare uno strumento per il #rifGlossario("diagramma di Gantt") che sia adatto alle nostre esigenze. Per questo motivo è stato deciso di proseguire la ricerca continuando nel frattempo ad utilizzare lo strumento fornito da #rifGlossario("GitHub").
+
+== Possibile sostituzione di Apache Nifi
+Durante il colloquio di SAL con l'azienda si è aperta la possibilità di sostituire #rifGlossario("Apache Nifi") con delle alternative, per sopperire ai suoi limiti riguardo l'interazione con i #rifGlossario("LLM"). Per questo _sprint_ si è deciso di studiare la fattibilità di costruire un servizio _custom_ di _data #rifGlossario("stream processing")_ utilzzando il _framework_ #rifGlossario("Spring").
+
+== Ampliamento delle Norme di Progetto
+Completata la prima stesura del Piano di Qualifica nell'ultimo _sprint_, ora si è reso necessario ampliare le Norme di Progetto compilando le sezioni relative alle procedure di verifica e validazione che verranno implementate nel progetto.
+
+== Aggiornamenti alle Norme di Progetto
+In aggiunta all'attività di ampliamento il gruppo ha fatto emergere la necessità di apportare alcune modifiche a delle sezioni già scritte, ovvero:
+- Aggiornare la sezione riguardante i processi organizzativi.
+- Aggiungere, nella sezione Processi Primari, le norme su come descrivere i casi d'uso.
+
+== Colloquio con il prof. Cardin
+Il gruppo ha deciso di organizzare un incontro con il prof. Cardin per discutere dei dubbi emersi durante la stesura dei casi d'uso. L'incontro è previsto per il 19 dicembre 2024.
+
+== Stesura di nuovi casi d'uso
+Continua la necessità di procedere con l'analisi dei requisiti. Nel prossimo _sprint_, in particolare, si prevede di soffermarsi principalmente sui casi d'uso aventi come attore principale l'amministratore del servizio di noleggio. Questa attività avrà una priorità più alta in termini di tempistiche, con l'obiettivo di arrivare al colloquio con il prof. Cardin sopracitato collezionando più dubbi possibili.
+
+== Affinamento dashboard di Grafana
+Per la fine del prossimo _sprint_ si prevede di avere una versione più affinata della dashboard, frutto anche di uno studio più approfondito di #rifGlossario("Grafana").
```

Comment:
In questo paragrafo, è da mettere in corsivo "dashboard" su titolo e al suo interno una volta.